### PR TITLE
Models: CRUD, Signal

### PR DESCRIPTION
**Pull request by IgorGolub13**
Implemented CRUD for CustomUser, Transaction, Category and Debt models. Added creating default categories for new users by post_save Signal.

### CODE REVIEW

File: src/core/models.py [hunk 1]
```diff
@@ -37,40 +38,37 @@ def __str__(self):
 
 
 class Category(models.Model):
+    user = models.ForeignKey(get_user_model(), related_name='category', on_delete=models.CASCADE)
     name = models.CharField(_('Ім\'я'), max_length=40)
-    description = models.TextField(_('Опис'), max_length=500)
+    description = models.TextField(_('Опис'), max_length=500, null=True, blank=True)
+    type = models.CharField(_('Тип'), choices=CategoryType.TYPE_CHOICES, max_length=1, default='-')
```
Comment (IvanYukish) [inline src/core/models.py:44]: DebtMode.LEND

File: src/templates/403.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+{% load i18n %}
+
+{% block head_title %}{% trans "Access is denied" %}{% endblock %}
+
+{% block content %}
+    <div class="d-flex justify-content-center">
+        <h1>{% trans "Access denied!" %}</h1>
+    </div>
+{% endblock %}
```
Comment (IvanYukish) [inline src/templates/403.html:11]: + line

File: src/user/signals.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+@receiver(post_save, sender=CustomUser)
+def create_fixture_category(sender, instance, created, **kwargs):
+    if created:
+        Category.objects.bulk_create(
+            [Category(user=instance, name=i[0], type=i[1]) for i in CATEGORY_FIXTURE])
+    else:
+        pass
+
+
+post_save.connect(create_fixture_category, sender=CustomUser)
```
Comment (IvanYukish) [inline src/user/signals.py:18]: line

Comment (IvanYukish) [inline src/user/signals.py:18]: check without connect save, may it isn't required.

File: src/user/urls.py [hunk 4]
```diff
@@ -1,8 +1,11 @@
...
 
-from user.views import ProfileDetailView
+from user.views import ProfileDetailView, ProfileUpdateView, UserListView
 
 urlpatterns = [
     path('', include('allauth.urls')),
     path('profile/', ProfileDetailView.as_view(), name='profile'),
+    path('profile/update', ProfileUpdateView.as_view(), name='profile-update'),
+
+    path('list/', UserListView.as_view(), name='user'),
```
Comment (IvanYukish) [inline src/user/urls.py:10]: name=user-list

File: src/user/views.py [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
-
-    def get_context_data(self, **kwargs):
-        pass
+    template_name = 'user/user_list.html'
+    context_object_name = 'users'
+    paginate_by = 10
 
     def get_queryset(self):
-        return self.queryset
+        return self.model.objects.all()
```
Comment (IvanYukish) [inline src/user/views.py:16]: it'll return all user by default

File: src/core/models.py (hunk 1 above)
Comment (IgorGolub13) [reply src/core/models.py]: Solved.

File: src/templates/403.html (hunk 2 above)
Comment (IgorGolub13) [reply src/templates/403.html]: Solved.

File: src/user/signals.py (hunk 3 above)
Comment (IgorGolub13) [reply src/user/signals.py]: Solved.

Comment (IgorGolub13) [reply src/user/signals.py]: Solved. It isn't required.

File: src/user/urls.py (hunk 4 above)
Comment (IgorGolub13) [reply src/user/urls.py]: Solved.

File: src/user/views.py (hunk 5 above)
Comment (IgorGolub13) [reply src/user/views.py]: Solved.

**Pull request merged**